src/services/pcn-iptables/src/Utils.cpp

```diff
@@ -608,7 +612,9 @@ void Chain::horusFromRulesToMap(
 
   for (auto const &rule : rules) {
     i++;
-    fromRuleToHorusKeyValue(rule, key, value);
+    if (i > HorusConst::MAX_RULE_SIZE_FOR_HORUS)
```

Comment:
Shouldn't this be `>=`?

Reply:
It should be `>`, but since I reworked the cycle to avoid confusion `>=` works now.